### PR TITLE
Iow 444

### PR DESCRIPTION
**Pull request by dsiemon2**
Before making a pull request
----------------------------

- [ x] Make sure all tests run
- [ x] Update the changelog appropriately
- [ x] Make sure the Swagger docs are accurate

Title
-----------
Available statistical time series service should return just the tsId as the id

Description
-----------
* The id element in the timeSeries array contains only the tsId
* Fix the swagger docs as described.

After making a pull request
---------------------------
- [ ] If appropriate, put the link to the PR in the JIRA ticket
- [ ] Assign someone to review unless the change is trivial

### CODE REVIEW

**Review by kkehl-usgs — APPROVED** [review]
Would be better to make the title of the PR more descriptive.  I usually copy the title from the JIRA story.

File: src/main/java/gov/usgs/wma/waterdata/openapi/schema/observations/StatisticalFeatureGeoJSON.java [hunk 1]
```diff
@@ -1,11 +1,11 @@
 package gov.usgs.wma.waterdata.openapi.schema.observations;
 
 import com.fasterxml.jackson.annotation.JsonPropertyOrder;
-
 import gov.usgs.wma.waterdata.openapi.schema.geojson.Geometry;
 import io.swagger.v3.oas.annotations.media.Schema;
 
 
+
```
Comment (orosz-usgs) [inline src/main/java/gov/usgs/wma/waterdata/openapi/schema/observations/StatisticalFeatureGeoJSON.java:8]: maybe being too picky, extra blank line

**Comment by dsiemon2** [comment]
https://internal.cida.usgs.gov/jira/browse/IOW-444

This is the title a copy from the jira..
I may have missed copying some of the description  (copy and paste)
I can update it

File: src/main/java/gov/usgs/wma/waterdata/openapi/schema/observations/StatisticalFeatureGeoJSON.java (hunk 1 above)
Comment (orosz-usgs) [reply src/main/java/gov/usgs/wma/waterdata/openapi/schema/observations/StatisticalFeatureGeoJSON.java:8]: line 7 could be deleted too

**Review by orosz-usgs — APPROVED** [review]
Just the picky blank lines

**Pull request merged**